### PR TITLE
Fixes : Support for recorded and paused time

### PR DESCRIPTION
**Pull request by FJBDev**
-  Added the ability for the user to choose the time value used to compute the pace (Recorded vs Moving)
image

- Tour Book view now uses the paused time value from TourData

- Supporting the display of pauses in the chart and 2d map for multiple tours

- Putting back my code that Wolfgang's merge in main removed

### CODE REVIEW

File: bundles/net.tourbook/src/net/tourbook/messages.properties [hunk 1]
```diff
@@ -1741,6 +1742,8 @@ Pref_Appearance_NumberOfRecent_TourTypes             = Number of recent tour t&y
 Pref_Appearance_NumberOfRecent_TourTypes_Tooltip     = Number of recently used tour types which are displayed in the context menu.\n\
                                                        \n\
                                                        0 will hide recently used tour types.
+Pref_Appearance_Radio_UseRecordedTime                = Recorded Time
+Pref_Appearance_Radio_UseMovingTime                  = Moving Time
```
Comment (wolfgang-ch) [inline bundles/net.tourbook/src/net/tourbook/messages.properties:1746]: no mnemonic

File: bundles/net.tourbook/src/net/tourbook/messages.properties [hunk 2]
```diff
@@ -1731,6 +1731,7 @@ Pref_Appearance_Check_AutoOpenTagging                = &Open "Add Tags..." autom
 Pref_Appearance_Check_TaggingAnimation               = Show &animation  \u00BB\u00BB\u00BB\u00BB   in the context menu
 Pref_Appearance_Dialog_ResetAllToggleDialogs_Message = All hidden dialogs will be displayed again.
 Pref_Appearance_Dialog_ResetAllToggleDialogs_Title   = Hidden Dialogs
+Pref_Appearance_Group_PaceDisplay                    = Pace Display Menu
 Pref_Appearance_Group_Tagging                        = Tagging Context Menu
```
Comment (wolfgang-ch) [inline bundles/net.tourbook/src/net/tourbook/messages.properties:1735]: I don't understand the name `Pace Display Menu`, is there a pace display menu?

File: bundles/net.tourbook/src/net/tourbook/messages.properties [hunk 3]
```diff
@@ -3462,6 +3465,8 @@ pref_appearance_number_of_recent_tags         = Number of recent &tags:
 pref_appearance_number_of_recent_tags_tooltip = Number of recently used tags which are displayed in the context menu.\n\
                                                 \n\
                                                 0 will hide recently used tags.
+pref_appearance_pace_computation_option         = Pace computation options:
```
Comment (wolfgang-ch) [inline bundles/net.tourbook/src/net/tourbook/messages.properties:3468]: A label should not have a colon

File: bundles/net.tourbook/src/net/tourbook/ui/views/tourBook/TVITourBookItem.java [hunk 4]
```diff
@@ -22,31 +22,34 @@
...
    public static final String SQL_ALL_TOUR_FIELDS;
 
    public static final String SQL_ALL_OTHER_FIELDS;
-   public static final int    SQL_ALL_OTHER_FIELDS__COLUMN_START_NUMBER;
 
+   public static final int    SQL_ALL_OTHER_FIELDS__COLUMN_START_NUMBER;
    static final String        SQL_SUM_COLUMNS;
-   static final String        SQL_SUM_FIELDS;
 
+   static final String        SQL_SUM_FIELDS;
```
Comment (wolfgang-ch) [inline bundles/net.tourbook/src/net/tourbook/ui/views/tourBook/TVITourBookItem.java:52]: code is reformatted

File: bundles/net.tourbook/src/net/tourbook/messages.properties (hunk 2 above)
Comment (FJBDev) [reply bundles/net.tourbook/src/net/tourbook/messages.properties:1735]: No, I will fix that

File: bundles/net.tourbook/src/net/tourbook/ui/views/calendar/CalendarProfileManager.java [hunk 5]
```diff
@@ -252,73 +254,72 @@
...
+   private static final DataFormatter   _weekFormatter_Time_Moving;
+   private static final DataFormatter   _weekFormatter_Time_Break;
+   static final DataFormatter[]         allTourContentFormatter;
+
+   static final DataFormatter[]         allWeekFormatter;
+   private static final IValueFormatter _valueFormatter_Number_1_0 = new ValueFormatter_Number_1_0(false);
 
 // SET_FORMATTING_OFF
 
-   private static final IValueFormatter   _valueFormatter_Number_1_0            = new ValueFormatter_Number_1_0(false);
```
[the remark this answers is not in the document]
Comment (FJBDev) [reply bundles/net.tourbook/src/net/tourbook/ui/views/calendar/CalendarProfileManager.java:314]: Arg, Eclipse.. I will fix that.

File: bundles/net.tourbook/src/net/tourbook/messages.properties (hunk 3 above)
Comment (FJBDev) [reply bundles/net.tourbook/src/net/tourbook/messages.properties:3468]: I will fix that.

File: bundles/net.tourbook/src/net/tourbook/ui/views/tourBook/TVITourBookItem.java (hunk 4 above)
Comment (FJBDev) [reply bundles/net.tourbook/src/net/tourbook/ui/views/tourBook/TVITourBookItem.java:52]: Arg, Eclipse !! I will fix that

File: bundles/net.tourbook/src/net/tourbook/messages.properties (hunk 1 above)
Comment (FJBDev) [reply bundles/net.tourbook/src/net/tourbook/messages.properties:1746]: I will add menmonics

File: bundles/net.tourbook/src/net/tourbook/messages.properties (hunk 2 above)
Comment (FJBDev) [reply bundles/net.tourbook/src/net/tourbook/messages.properties:1735]: Fixed

File: bundles/net.tourbook/src/net/tourbook/messages.properties (hunk 3 above)
Comment (FJBDev) [reply bundles/net.tourbook/src/net/tourbook/messages.properties:3468]: Fixed. And I also fixed the label `pref_appearance_number_of_recent_tags` above as it has also a colon

File: bundles/net.tourbook/src/net/tourbook/messages.properties (hunk 1 above)
Comment (FJBDev) [reply bundles/net.tourbook/src/net/tourbook/messages.properties:1746]: Fixed

File: bundles/net.tourbook/src/net/tourbook/ui/views/calendar/CalendarProfileManager.java (hunk 5 above)
[the remark this answers is not in the document]
Comment (FJBDev) [reply bundles/net.tourbook/src/net/tourbook/ui/views/calendar/CalendarProfileManager.java:314]: Fixed in my current PR

File: bundles/net.tourbook/src/net/tourbook/ui/views/tourBook/TVITourBookItem.java (hunk 4 above)
Comment (FJBDev) [reply bundles/net.tourbook/src/net/tourbook/ui/views/tourBook/TVITourBookItem.java:52]: Fixed in my current [PR](https://github.com/wolfgang-ch/mytourbook/pull/233)

**Pull request merged**